pkgs/development/libraries/gfbgraph/default.nix

```diff
@@ -1,23 +1,35 @@
 { stdenv, fetchurl, pkgconfig, glib, librest, gnome-online-accounts
-, gnome3, libsoup, json-glib, gobject-introspection }:
+, gnome3, libsoup, json-glib, gobject-introspection
+, gtk-doc, pkgs, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "gfbgraph";
-  version = "0.2.3";
+  version = "0.2.4";
 
-  outputs = [ "out" "dev" "devdoc" ];
```

Comment:
This will remove the documentation. We probably need other dependencies (`docbook-xsl-nons` and maybe some DTDs) now that they did not include prebuilt documentation in the tarball.

Reply:
I removed `devdoc` because I could not get it to build... it throws the following error:

```
checking for references to /build/ in /nix/store/0fg896bpjm61n95ja4089fqc45ijax36-gfbgraph-0.2.4-dev...
find: '/nix/store/b3mhsnabddfygv6f9xyxdkwxrr02rr23-gfbgraph-0.2.4-devdoc': No such file or directory
strip is /nix/store/a759kbr0kgccsxl6lfx0xv5i3c9x9bw9-binutils-2.31.1/bin/strip
builder for '/nix/store/0n25brpxnk8x9g3aq67ljqvpagij4zq3-gfbgraph-0.2.4.drv' failed to produce output path '/nix/store/b3mhsnabddfygv6f9xyxdkwxrr02rr23-gfbgraph-0.2.4-devdoc'
cannot build derivation '/nix/store/ajqisggzn8pwz1li9xycwl8nm8chbxrk-gnome-maps-3.36.4.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/ykaln8iqk7hpm2h6a8i6gr05jwnvy242-gnome-online-miners-3.34.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/2lkyl5s89mrbp082a6qs4hpdpdib4qi4-gnome-photos-3.34.2.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/bmbswa1nvjkfvasq46lfxif042d6skdk-dbus-1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/bmrznddyf2y5zmsiw22svyqc7zc0ykhi-gnome-maps-3.36.4_fish-completions.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/636pggipwn0rk4i4ppnnvah5ma0whjm3-gnome-online-miners-3.34.0_fish-completions.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/k91va7hnj6vizdvy6ib0r94jyprb550d-gnome-photos-3.34.2_fish-completions.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/9xwrvyhvqjvbjv3inyq2hqdfbvb4vblp-man-paths.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/i4254p6qnp21ib583qmgnxb5nq3xk2dn-system-path.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/9gz4is2jwxpqa1sdv2jgxaqrgmnzkpyr-nixos-system-snares-20.09.git.397f98f60a9M.drv': 1 dependencies couldn't be built
error: build of '/nix/store/9gz4is2jwxpqa1sdv2jgxaqrgmnzkpyr-nixos-system-snares-20.09.git.397f98f60a9M.drv' failed
```
I do not know how to fix this

Reply:
Yeah, that would be because gtk-doc documentation is not being built. The error is probably above. Or maybe it needs `--enable-gtk-doc` in `configureFlags`.

Reply:
@jtojnar You were right. It needed the `--enable-gtk-doc` flag. Now it builds.